.github/workflows/test.yml

```diff
@@ -19,3 +19,14 @@ jobs:
 
     - name: Run Unit Tests
       run: npm run test
+
+    - name: Run E2E Tests
```

Comment:
could you add instructions somewhere to run this locally as well?

Reply:
Sure, I added npm scripts to we can run it locally and README is also updated.